requirements.dev.txt

```diff
@@ -1,5 +1,7 @@
 # Packages required for development and committing to repo.
 
 # Formatting packages pinned for consistency with format checker.
+ruff == 0.5.5
```

Comment:
nit: can you change lines 6-7 to match your style here (or change 4 and 5 to match).